go/client/logclient.go

```diff
@@ -226,51 +252,43 @@ func (c *LogClient) addChainWithRetry(ctx context.Context, ctype ct.LogEntryType
 	for _, link := range chain {
 		req.Chain = append(req.Chain, link)
 	}
-	httpStatus := "Unknown"
-	// Retry after 1s, 2s, 4s, 8s, 16s, 32s, 64s, 128s, 128s, ....
-	maxInterval := 128.0
+	// Retry after 1s, 2s, 4s, 8s, 16s, 32s, 64s, 128s, 128s
 	backoffInterval := 1.0
 	backoffSeconds := 0
 loop:
 	for {
 		if backoffSeconds > 0 {
-			log.Printf("Got %s, backing-off %d seconds", httpStatus, backoffSeconds)
-		}
-		err := backoffForRetry(ctx, time.Second*time.Duration(backoffSeconds))
-		if err != nil {
-			return nil, err
-		}
-		if backoffSeconds > 0 {
+			err := backoffForRetry(ctx, time.Second*time.Duration(backoffSeconds))
```

Comment:
`backoffForRetry` also handles `ctx.Done()`, so I think this might change behaviour for immediate retries (e.g. 408)